src/main/java/seedu/address/model/applicant/Status.java

```diff
@@ -0,0 +1,26 @@
+package seedu.address.model.applicant;
+
+/**
+ * Represents an Applicant's status in the address book.
+ * default value is "Available" and is replaced with a position name when Applicant accepts an offer
+ */
+public class Status {
+    private final String value;
+
+    /**
+     * Constructs a {@code HiredStatus}.
+     * Initialised as available as every Applicant added to HireLah is applying for a position.
+     */
+    public Status() {
+        value = "Available";
+    }
+
+    public Status(String positionName) {
+        value = positionName;
```

Comment:
Maybe the value of the status can be something like `"Hired as" + positionName` for better clarify?

Reply:
Good points. I think `ApplicantStatus` might be still be abit vague so i decided to rename the class to `HiredStatus` as the main purpose of this status is to mark if the applicant has been hired or not as a result of obtaining the job offer.

Also agreed on adding a preamble of "Hired as" to the value for better clarity